melodic/distribution.yaml

```diff
@@ -5237,6 +5237,26 @@ repositories:
       url: https://github.com/peci1/movie_publisher.git
       version: melodic-devel
     status: developed
+  mpc_local_planner:
+    doc:
+      type: git
+      url: https://github.com/rst-tu-dortmund/mpc_local_planner.git
```

Comment:
It looks like the source repository is missing a copy of the COPYING file from the GPLv3.  I think once that is added, this should be ready to go.

Reply:
Oh yes, done :)